data/txHandler_test.go

```diff
@@ -1433,21 +1585,31 @@ func runHandlerBenchmarkWithBacklog(maxGroupSize, tps int, invalidRate float32,
 		invalidCounter := 0
 		defer func() {
 			if groupCounter > 1 {
+				timeSinceStart := time.Since(tt)
 				droppedBacklog, droppedPool := getDropped()
-				b.Logf("Input T(grp)PS: %d (delay %f microsec)", tps, float64(rateAdjuster)/float64(time.Microsecond))
-				b.Logf("Verified TPS: %d", uint64(txnCounter)*uint64(time.Second)/uint64(time.Since(tt)))
-				b.Logf("Time/txn: %d(microsec)", uint64((time.Since(tt)/time.Microsecond))/txnCounter)
+				if tps > 0 {
+					b.Logf("Input T(grp)PS: %d (delay %f microsec)", tps, float64(rateAdjuster)/float64(time.Microsecond))
+				}
+				b.Logf("Verified TPS: %d T(grp)PS: %d", uint64(txnCounter)*uint64(time.Second)/uint64(timeSinceStart),
```

Comment:
nit:

Why is the TPS calculation written as
`count` x `1/window` ?
since TPS is a value-per-second, i think it reads better as
`txnCounter / (timeSinceStart / time.Second)`
^transactions ^per ^(how many seconds in the window)

Again, super minor nit-pick, no worries if you don't take this one :)

Reply:
straightforward math without parenthesis :-)